include/deal.II/base/utilities.h

```diff
@@ -72,6 +75,32 @@ namespace Utilities
   std::string
   dealii_version_string();
 
+  /**
+   * Assign to each point in @p points an index using Hilbert space filling curve.
+   * To that end, bounding box for @p points will be determined, based on which their
+   * integer coordinates are calculated.
+   * The linear index is given as a dim-collection of bits, from high to low.
+   *
+   * The depth of the Hilbert curve (i.e. the number of bits per dimension) by
+   * default is equal to <code>std::numeric_limits<Integer>::digits/code>. The
+   * function is instantiated for <code>unsigned long int</code> and
+   * <code>double</code>.
+   */
+  template <typename Integer, int dim, typename Number>
+  std::vector<std::array<Integer, dim>>
+  inverse_Hilber_space_filling_curve(
```

Comment:
wrong spelling of the name in the name of the function :-)

Reply:
oops ;-)